### PR TITLE
[iOS] Fix BackgroundColor issue with Frame

### LINKED ISSUE #11031: [Bug] Regression in 4.7-pre4: Frames are broken
### Description

Xamarin.Forms 4.7-pre4 breaks frames by removing clipping from them, making the background color bleed to fill the control's dimensions. Frames do work in 4.7-pre3. Edit: This happens on iOS only.

Pre-3:
Pre-3

Pre-4:
Pre-4

### Steps to Reproduce

The following snippet will work in pre-3 but not in pre-4.
```xaml
                <Frame BackgroundColor="Red" BorderColor="Blue" CornerRadius="20">
                    <Label HorizontalTextAlignment="Center" Text="HELLO" />
                </Frame>
```

Adding IsClippedToBounds="True" makes no difference.

- Version with issue: 4.7.0.937-pre4
- Last known good version: 4.7.0.848-pre3
- IDE: VS For Mac 8.7 Preview, build 1077

### PR DESCRIPTION
**Pull request by jsuarezruiz**
### Description of Change ###

Fix `BackgroundColor` issue with Frame on iOS.

Caused by #11001 and #10469

### Issues Resolved ### 

- fixes #11031

### API Changes ###
 
 None

### Platforms Affected ### 

- iOS

### Behavioral/Visual Changes ###

None

### Before/After Screenshots ### 
fix11031

### Testing Procedure ###
Launch Core Gallery and navigate to the issue 11031. Verify that the Frame BackgroundColor is clipped by the CornerRadius.

### PR Checklist ###

- [ ] Targets the correct branch
- [ ] Tests are passing (or failures are unrelated)

### CODE REVIEW

**Comment by Tommigun1980** [comment]
Woohoo! Fantastic velocity, thanks. Looking forward to the next release 👍

File: Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs [hunk 1]
```diff
@@ -79,7 +79,13 @@ public virtual void SetupLayer()
 			if (Element.BackgroundColor == Color.Default)
 				_actualView.Layer.BackgroundColor = ColorExtensions.BackgroundColor.CGColor;
 			else
+			{
+				// BackgroundColor gets set on the base class too which messes with
+				// the corner radius, shadow, etc. so override that behaviour here
+				BackgroundColor = null;
```
Comment (samhouts) [inline Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs:85]: hmm, this was causing other tests to fail, though

Comment (samhouts) [reply Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs:85]: that's why i removed it!

Comment (jsuarezruiz) [reply Xamarin.Forms.Platform.iOS/Renderers/FrameRenderer.cs]: Oh, ok!. I changed it. The BackgroundColor should work but I removed this line.

**Pull request merged**